<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managebook</class>
 <widget class="QDialog" name="managebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="edit_book_table">
     <property name="title">
      <string>Chỉnh sửa sách</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Tìm kiếm</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="input_search"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_search">
              <property name="text">
               <string>Tìm kiếm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>Thao tác</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="btn_add_book">
              <property name="text">
               <string>Thêm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_update_book">
              <property name="text">
               <string>Cập nhật</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_edit_book">
              <property name="text">
               <string>Sửa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_reset_book">
              <property name="text">
               <string>Nhập lại</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="input_book_issuing_company">
           <property name="title">
            <string>Thông tin</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Id</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="input_book_id">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Tiêu đề</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="input_book_title"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Tổng số lượng</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="input_book_total"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Có sẵn</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="input_book_available"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Giá</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="input_book_price"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Loại bìa</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="input_book_cover"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Ngày xuất bản</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QDateEdit" name="input_book_publication"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Nhà xuất bản</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLineEdit" name="input_book_publisher">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_change__book_publisher">
                 <property name="text">
                  <string>Chỉnh sửa</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Chuyên mục</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLineEdit" name="input_book_category">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_change_book_category">
                 <property name="text">
                  <string>Chỉnh sửa</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Cty phát hành</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLineEdit" name="input_book_issuing_company_2">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_change_book_issuing_compan">
                 <property name="text">
                  <string>Chỉnh sửa</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Tác giả</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QListView" name="listView"/>
               </item>
               <item>
                <widget class="QListView" name="listView_2"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_11">
        <property name="title">
         <string>Bảng dữ liệu</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableView" name="table_book"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
